src/System.Globalization/ref/System.Globalization.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Globalization
 {
+    // ToDo: Calendar should be removed from here once it is available in System.Runtime
```

Comment:
I'm basically tracking it with #10075 which won't be closed until all of the ToDo's are done. I know that P2P would work, but instead of working around that, I'm going to work from System.Runtime my way up, so that I don't have things like this.